10up/simple-google-news-sitemap
# add plugin assets

### Description of the Change
This PR adds in the banner and icon assets for rendering on WP.org (as well as within the WP Admin/plugin interface).


### How to test the Change
Build plugin, install on test site, browse to active plugin page and click view details on plugin to see image render

### Changelog Entry

> Added - Plugin banner and image assets


### Credits

Props [Brooke Campbell](https://www.linkedin.com/in/brookecampbelldesign/)


### Checklist:
